packages/wrangler/src/config/validation-pages.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * Pages now supports configuration via `wrangler.toml`. As opposed to
+ * Workers however, Pages only supports a limited subset of all available
+ * configuration keys.
+ *
+ * This file contains all `wrangler.toml` validation things, specific to
+ * Pages.
+ */
+
+import { FatalError } from "../errors";
+import { defaultWranglerConfig } from "./config";
+import { Diagnostics } from "./diagnostics";
+import type { Config } from "./config";
+
+const supportedPagesConfigFields = [
+	"pages_build_output_dir",
+	"name",
+	"compatibility_date",
+	"compatibility_flags",
+	"send_metrics",
+	"limits",
+	"placement",
+	"vars",
+	"kv_namespaces",
+	"queues", // `producers` ONLY
+	"r2_buckets",
+	"d1_databases",
+	"vectorize",
+	"hyperdrive",
+	"services",
+	"analytics_engine_datasets",
+	"ai",
+	"dev",
+] as const;
```

Comment:
OH! nice catch! I was 100% convinced we won't support DOs, which is why they were never in here. My audit doc is also wrong, so will update as well

Reply:
fixed!

Reply:
I just realized that if we support DOs, we need to support `migrations` as well. Will need to add

Reply:
No, because we'll only be supporting DOs defined in a different worker (AFAIK this is the current Pages status quo?), and I'm pretty sure you can't define cross worker DO migrations.

Reply:
we're gonna run into some warnings then like: https://github.com/cloudflare/workers-sdk/pull/5234/files#diff-91e21fab7c621dcd160ff3a7731adefac5ef37bd372fce4b7c51f5f4393c313cR38-R46. Those will be very noisy AFAICT. Are we OK with that?

Reply:
great, then no other necessary changes need wrt DOs